src/main/java/org/jolokia/docker/maven/model/ContainerDetails.java

```diff
@@ -62,6 +63,18 @@ public String getName() {
     }
 
     @Override
+    public String getIPAddress() {
+        if (json.has(NETWORK_SETTINGS) && !json.isNull(NETWORK_SETTINGS)) {
+            JSONObject networkSettings = json.getJSONObject(NETWORK_SETTINGS);
+            if (!networkSettings.isNull(PORTS)) {
```

Comment:
Shouldn't we check here for `IP` ? Returning a null IP address when no ports are configured although there would be one seems not so intuitive.